packages/utils/formatNumber.ts

```diff
@@ -58,10 +71,57 @@ export function formatNumber(
   return `${limitIndicator}${valueToDisplay}`
 }
 
+export function formatFiatNumber({
```

Comment:
It's better that we treat `formatFiatNumber` as formatting numbers with a specific rule which suits fiat display. And `formatNumber` is the base function that allow us to specify the rule and `formatFiatNumber` should call `formatNumber` and specify the rule it needs

Reply:
resolved these. lets chat monday